src/Button/Button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Please use the following structure of the project:
- srr
- - components
- - - Button.js
- - - Display.js
- - App.js